pom.xml

```diff
@@ -146,6 +146,12 @@ under the License.
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-annotations</artifactId>
```

Comment:
What do you need jackson-annotattions for?
I dont think see them being used anywhere in this PR,